docs/walrus-sites/restrictions.md

```diff
@@ -53,3 +53,7 @@ The connection with the Sui Wallet apps works on Android devices.
 This limitation only applies to Portal based on service workers. A web Portal will not
 have this limitation.
 ```
+
+## No support for progressive web apps (PWAs)
+
+With the current design, Walrus Sites cannot be used for progressive web apps (PWAs).
```

Comment:
Suggestion: added a couple of lines explaining the reasons why this is not supported. Please discard if unnecessary.

```suggestion
With the current design, Walrus Sites cannot be used for progressive web apps (PWAs).
Two characteristics of the service-worker portal prevent support for PWAs:

- Since the service worker needs to be registered for the page to work, the PWA's manifest file cannot be loaded by the browser directly;
- There can only be one service worker registered per origin. Therefore, registering a PWA's service worker would remove the Walrus Sites service worker, breaking Walrus Sites' functionality.

Note that the server-side portal does share these limitations. However, for the moment, we support both technologies: Walrus Sites must be able to load from both a service-worker portal and a server-side portal, and therefore have to be built with the more restrictive feature set. For more details, see the [installation requirements for PWAs](https://en.wikipedia.org/wiki/Progressive_web_app#Installation_criteria).